.github/workflows/test.yml

```diff
@@ -23,10 +23,38 @@ jobs:
         run: |
           pip install -U pip setuptools wheel
           pip install -r ./requirements.txt
-
       - name: Run pre-commit
         run: pre-commit install && pre-commit run --all-files
 
+  reapply:
+    runs-on: ubuntu-latest
+    continue-on-error: true
+    strategy:
+      matrix:
+        python-version: ['3.9']
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 2
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v3
+        with:
+          python-version: ${{ matrix.python-version }}
+      - name: Install dependencies
+        run: |
+          pip install -U pip setuptools wheel
+          pip install -r ./requirements.txt
+          pip install 'Django>=3.2,<4.0' isort mypy types-pytz types-PyYAML types-docutils types-geoip2
+      - name: Check against Django code
+        run: ./scripts/compare_to_master.sh
+      - name: Save cache as notes
+        run: git -c 'user.email=django_stubs_test@example.com' -c 'user.name=Cache writing bot' notes --ref cache_history add -f -F .custom_cache/.apply_errors $cur_hash
+      - name: Push notes
+        uses: ad-m/github-push-action@master
+        with:
+          github_token: ${{ secrets.GITHUB_TOKEN }}
+          branch: refs/notes/cache_history
```

Comment:
Do we have any other options? Pushing to a branch seems like not the best idea to store cache 🤔